frontend/src/components/aepps-list.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-const list = [
+const aepplist = [
```

Comment:
camelCase naming convention for variables
e.g. aeppList, personList, firstName, lastName...